OpenOversight/app/models/database.py

```diff
@@ -334,21 +351,6 @@ def total_documented_officers(self) -> int:
             db.session.query(Officer).filter(Officer.department_id == self.id).count()
         )
 
-    def remove_database_cache_entries(self, update_types: List[str]) -> None:
-        """Remove the Department model key from the cache if it exists."""
-        remove_database_cache_entries(self, update_types)
```

Comment:
I think it's fine to keep this, then you don't have to change all the calls of that function

Reply:
They've all been changed already, it's more work to undo it than otherwise. I also think there's some value to it being more uniform.